apis/mysql/v1alpha1/databse_types.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
It looks like there's a typo in this filename - `databse` rather than `database`.

Reply:
Good catch, thanks 👍